docs/Release.md

```diff
@@ -0,0 +1,35 @@
+# Upstream Release Process
+
+## Tag dependent apbs
+
+- keycloak https://github.com/feedhenry/keycloak-apb
+- 3scale https://github.com/feedhenry/3scale-apb
+- sync https://github.com/feedhenry/fh-sync-server-apb
+- digger (mobile ci cd) https://github.com/feedhenry/aerogear-digger-apb  
+
+
+Run the following command in each of the above repos:
+
+```bash
+TAG=0.0.4 #example version
+git checkout $(TAG)
+make build_and_push TAG=$(TAG)
```

Comment:
the "docker_push" is pointless, no ? 

on the actual APBs we need a release process (script), like:

```
VERSION=x.y.z
git tag -s VERSION  sha1_of_commit_that_should_be_used -m "gpg signature"
git push origin VERSION
```


since the Docker image is build based on changes in the repo

Reply:
sure will remove

Reply:
irgnore what I said